src/python/setup.py.in

```diff
@@ -48,10 +48,12 @@ ext_modules = cythonize(ext_modules)
 
 for module in pybind11_modules:
     my_include_dirs = include_dirs + [pybind11.get_include(False), pybind11.get_include(True)]
-    if module == 'hera':
+    if module == 'hera/wasserstein':
         my_include_dirs = ['@HERA_WASSERSTEIN_INCLUDE_DIR@'] + my_include_dirs
+    elif module == 'hera/bottleneck':
+        my_include_dirs = ['@HERA_BOTTLENECK_INCLUDE_DIR@'] + my_include_dirs
```

Comment:
```suggestion
    my_include_dirs = include_dirs + [pybind11.get_include(False), pybind11.get_include(True), '@HERA_WASSERSTEIN_INCLUDE_DIR@', '@HERA_BOTTLENECK_INCLUDE_DIR@']
```
Works fine too. It is up to you.

Reply:
There are files with the same name and different content in the directories of bottleneck and wasserstein, so it is on purpose that I only include them where needed.

Reply:
Ok, so forgot my comment.